phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
```

Comment:
Why not? See #2562 please. Comments @core23 commented on 12 Jan

`Described in my description text for this PR. If my php.ini uses a precision of 12, the already written unit tests will fail otherwise.`